app/controllers/hiring_staff/vacancies/vacancy_publish_feedback_controller.rb

```diff
@@ -1,7 +1,8 @@
 class HiringStaff::Vacancies::VacancyPublishFeedbackController < HiringStaff::Vacancies::ApplicationController
+  before_action :set_vacancy, only: %i[new create]
```

Comment:
This actually already happens on the application controller, so you should be ok to remove it